packages/react-devtools-shared/src/devtools/views/DevTools.js

```diff
@@ -127,6 +127,33 @@ export default function DevTools({
     [enabledInspectedElementContextMenu, viewAttributeSourceFunction],
   );
 
+  const devToolsRef = useRef<HTMLElement | null>(null);
+
+  useEffect(() => {
+    if (devToolsRef.current === null) return;
+
+    const ownerWindow = devToolsRef.current.ownerDocument.defaultView;
+
+    if (showTabBar) {
+      ownerWindow.addEventListener('keydown', (event: KeyboardEvent) => {
+        if (event.ctrlKey || event.metaKey) {
+          switch (event.key) {
+            case '1':
+              setTab(tabs[0].id);
+              event.preventDefault();
+              event.stopPropagation();
+              break;
+            case '2':
+              setTab(tabs[1].id);
+              event.preventDefault();
+              event.stopPropagation();
+              break;
+          }
+        }
+      });
+    }
```

Comment:
Looks like you removed the cleanup function by accident (where you remove the event listener).

Reply:
> nit: Let's move this check to the start of the effect? No point to define `handleKeyDown` (or return a cleanup function) if we're not going to use it. smile

Sorry I don't understand. I thought we don't need the return statement.
Should I return an empty function?
```js 
return ()=>{};
```

Reply:
Before, you were always returning a "cleanup function" (removing an event listener) even if you weren't attaching it. I suggested you move the bailout check (if `!showTabBar`) earlier to avoid doing any of that work.

But in the case that you are adding an event listener, you should return a cleanup function that removes it.

Reply:
Oh my bad. I hope now everything is fine.